tests/lean/interactive/focus.lean

```diff
@@ -4,7 +4,7 @@ begin
   focus {  subst a
        --^ "command":"info"
          },
-     --^ "command":"info"
+       --^ "command":"info"
```

Comment:
@leodemoura A slight regression re. goal info. For doc support while typing a tactic, we want the trailing whitespace to logically belong to the tactic, whereas previously it belonged to the end separator so that they share the same goal position. I don't think there's an easy way to reconcile these conflicting demands.